test/jdk/ProblemList-Xcomp.txt

```diff
@@ -28,3 +28,4 @@
 #############################################################################
 
 java/lang/invoke/MethodHandles/CatchExceptionTest.java 8146623 generic-all
+java/lang/ScopedValue/StressStackOverflow.java 8308609 generic-x64
```

Comment:
java/lang/ScopedValue/StressStackOverflow.java#no-vmcontinuations ?

Reply:
Nope. This is the JDK21 version of the test and not the JDK22 version of the test.